src/redux/slices/debug-options-slice.ts

```diff
@@ -0,0 +1,93 @@
+import { createSlice, PayloadAction } from "@reduxjs/toolkit";
+import {
+  DebugOptions,
+  TileColoredBy,
+  BoundingVolumeColoredBy,
+  BoundingVolumeType,
+} from "../../types";
+import { RootState } from "../store";
+
+// Define a type for the slice state
+interface debugOptionsState {
+  value: DebugOptions;
+}
+const initialState: debugOptionsState = {
+  value: {
+    // Show minimap
+    minimap: false,
+    // Use separate traversal for the minimap viewport
+    minimapViewport: false,
+    // Show bounding volumes
+    boundingVolume: false,
+    // Select tiles with a mouse button
+    pickable: false,
+    // Load tiles after traversal.
+    // Use this to freeze loaded tiles and see on them from different perspective
+    loadTiles: false,
```

Comment:
Looks like loadTiles state should be set to true by default

Reply:
1. defined loadTiles as true in the debug-option-slice.ts